Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -675,8 +675,17 @@ function Main() {
 	Run_IMB_MPI1
 	Run_IMB_RMA
 	Run_IMB_NBC
-	Run_IMB_P2P
-	Run_IMB_IO
+	# Sometimes IMB-P2P and IMB-IO aren't available, skip them if it's the case
```

Comment:
what is the case "not available"? Those files are compiled from Intel Benchmark repo not MPI, and should be available all the times.

Reply:
SLES 12 SP4, INFINIBAND-INTEL-MPI-2VM TC. P2P and IO are missing

Reply:
acked. I recall that.